share/spice/sports/mlb/games/sports_mlb_games.js

```diff
@@ -15,7 +15,7 @@
             return Spice.failed('mlb_games');
         }
         
-        DDG.require('sports', function(){
+        DDG.require(['moment.js', 'sports'], function(){
```

Comment:
The call to Games.init(MLBInit); in this function callback also requires moment.js, so including that here to speed things up.